lib/ansible/plugins/callback/unixy.py

```diff
@@ -0,0 +1,203 @@
+# (c) 2012-2014, Michael DeHaan <michael.dehaan@gmail.com>
+#
```

Comment:
You might want to update this to our new format: http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#copyright